packer/README.md

```diff
@@ -34,6 +34,58 @@ vagrant box remove centos7-katello-devel-stable
 sudo virsh vol-delete --pool default centos7-katello-devel-stable_vagrant_box_image_0.img
 ```
 
+### Creating version specific dev boxes
+
+To create a version specific dev box, simply:
+
+```
+cd ./forklift/packer
+```
+
+and run this comand (replacing 3.18 with the correct version):
+
+```
+./scripts/build_stable_dev_box.rb 3.18
+```
+
+To test the newly created box:
+1.  add the printed box definition to vagrant/boxes.d/99-local.yaml 
+2.  run printed 'vagrant box add' command
+3.  vagrant up centos7-katello-3.18-stable
+4.  Verify that 'rpm -q katello-repos' is correct and the foreman and katello git repos are on the correct branches
+
+To upload the box:
+1. While you can try with the latest version of vagrant, versions 2.2.11-2.2.14 cannot properly upload boxes.  Download 2.2.9 (this will not interfere with your rpm installed vagrant):
+```
+wget https://releases.hashicorp.com/vagrant/2.2.9/vagrant_2.2.9_linux_amd64.zip
+unzip vagrant_2.2.9_linux_amd64.zip
+sudo mv vagrant /usr/bin/vagrant-upstream
+```
+2. Run  'vagrant-upstream login' and login with your credentials
+3. Ensure you are an owner of the katello organization in vagrnat cloud
+3. Use the command from the build_stable_dev_box.rb output to publish the box, it will look like:
+```
+vagrant-upstream cloud publish -d "katello-devel 3.18" -s "katello-devel 3.18" katello/katello-devel 3.18.0 libvirt centos7-katello-3.18-stable.box
+```
+
+NOTE: YOU WILL GET AN ERROR, BUT THE PUBLISH WILL ACTUALLY WORK
```

Comment:
Maybe a little more about this, `You will get an error (snippet from error message), this only happens during large box uploads to vagrant cloud and is a known vagrant issue, but the publish will actually work`